src/main.rs

```diff
@@ -171,37 +171,11 @@ fn get_pants_process() -> Result<Process> {
             (None, None, None, false)
         };
 
-    let env_pants_sha = env_version("PANTS_SHA")?;
     let env_pants_version = env_version("PANTS_VERSION")?;
-    if let Some(pants_sha) = &env_pants_sha {
```

Comment:
What do you think about landing this as a separate pull request? It'd at least help review by making this one smaller.